internal/packetfabric/cloud_service_aws.go

```diff
@@ -167,6 +167,8 @@ type CloudSettings struct {
 	PortCrossConnectOcid           string       `json:"port_cross_connect_ocid,omitempty"`
 	PortID                         string       `json:"port_id,omitempty"`
 	PublicIP                       string       `json:"public_ip,omitempty"`
+	PrimaryPublicIP                string       `json:"primary_public_ip,omitempty"`
```

Comment:
Assuming those 2 attributes will be added in the API? (I don't see them in https://docs.dev.packetfabric.net/api/v2/swagger/#/Cloud%20Router%20Connections/cloud_routers_connection)

Just to confirm the `v1.6.1` release will happen after those attributes will be added in PF.

Reply:
They are already present (and always have been for Azure). It just looks like they're undocumented in the docs.